packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_item_details/blocs/item_details_bloc.dart

```diff
@@ -58,11 +60,13 @@ class ItemDetailsBloc extends $ItemDetailsBloc {
   Stream<bool> _mapToIsLoadingState() => loadingState;
 
   @override
-  Stream<Result<ItemModel>> _mapToItemState() => _itemSubject;
-
-  @override
-  void dispose() {
-    _itemSubject.close();
-    super.dispose();
-  }
+  Stream<Result<ItemModel>> _mapToItemState() => Rx.merge([
```

Comment:
Instead of using `shareReplay(maxSize: 1)` on both places you can just use it once after as follows
`Rx.merge([...]).shareReplay(maxSize: 1)`

Reply:
done